chain/chunks/src/lib.rs

```diff
@@ -511,12 +512,11 @@ impl ShardsManager {
 
     pub fn check_chunk_complete(
         data_parts: usize,
-        total_parts: usize,
         chunk: &mut EncodedShardChunk,
+        rs: &ReedSolomon<reed_solomon_erasure::galois_8::Field>,
```

Comment:
Let's give this type some alias in `primitives`. Like `NightshadeReedSolomon` or something.